test/sjcl-ecdsa-recoverablepublickey-test.js

```diff
@@ -43,7 +43,7 @@ describe('ECDSA signing with recoverable public key', function(){
 
         var pub_val_point = secret_key._curve.G.mult(secret_key._exponent);
         var public_key = new sjcl.ecc.ecdsa.publicKey(curve, pub_val_point);
-        var hash = hash_function(message);
+        var hash = sjcl.bitArray.clamp(hash_function(message), secret_key._curveBitLength);
```

Comment:
I noticed that the blobvault messaging code uses sha512. The blobvault code (message.js and others) lives in ripple-lib even though it isn't required by anything in ripple-lib.